src/main/java/org/jboss/arquillian/testcontainers/api/Testcontainer.java

```diff
@@ -16,12 +16,12 @@
 
 /**
  * Used to annotate a field which <strong>must</strong> be an instance of a {@link GenericContainer}. A
- * {@link DockerRequired} annotation must be present on the type to use Testcontainer injection.
+ * {@link TestcontainersRequired} annotation must be present on the type to use Testcontainer injection.
  *
  * <pre>
  * &#064;ExtendWith(ArquillianExtension.class)
  * &#064;RunAsClient
- * // By throwing the TestAbortedException, the test will be skipped if docker is not available
+ * // By throwing the TestAbortedException, the test will be skipped if a container engine is not available
```

Comment:
This is a very minor comment. Should this "a Testcontainer engine"? I don't have a real strong opinion either way.

Reply:
I also think this needs a different wording, but I am not yet sure what's ideal wording is :)

Reply:
I'm fine to leave it as is, but no strong opinion either. I think, though, that the "engine" is either Docker or Podman (at this point), whereas Testcontainers is the... orchestration tool that starts a "container" using whatever "engine" is available in the environment.

Reply:
> I also think this needs a different wording, but I am not yet sure what's the ideal wording :)

Well, it seems that whole Javadoc is pretty outdated anyway. That sample code is pretty rough. Let me revisit that...

Reply:
Though "Testcontainer engine" is not a thing, how about going the descriptive way that "the test will be skipped if Testcontainers requirements are not available."

Reply:
@rhusar  @jamezp, Updated the sample code and just removed the comment altogether.